packages/core/src/amazonqFeatureDev/session/session.ts

```diff
@@ -285,6 +285,25 @@ export class Session {
         return { leftPath, rightPath, ...diff }
     }
 
+    public async sendMetricDataTelemetry(operationName: string, result: string) {
+        await this.proxyClient.sendMetricData({
+            metricName: 'Operation',
+            metricValue: 1,
+            timestamp: new Date(Date.now()),
```

Comment:
I think this should be good enough
```suggestion
            timestamp: new Date(),
```